lib/ruby_snowflake/client.rb

```diff
@@ -223,7 +223,7 @@ def raise_on_bad_response(response)
       def retryable_http_response_code?(code)
         # retry (in order): bad request, forbidden (token expired in flight), method not allowed,
         #   request timeout, too many requests, anything in the 500 range (504 is fairly common)
-        [400, 403, 405, 408, 429, 504].include?(code.to_i) || (500..599).include?(code)
+        [400, 403, 405, 408, 429, 502, 504].include?(code.to_i) || (500..599).include?(code.to_i)
```

Comment:
@alexstoick is this the same as
```ruby
[400, 403, 405, 408, 429].include?(code.to_i) || (500..599).include?(code.to_i)
```
? i.e. is it redundant to have `502` and `504` in the first list?

Reply:
made a quick PR: https://github.com/rinsed-org/rb-snowflake-client/pull/46